test/test_profiler.py

```diff
@@ -134,6 +135,31 @@ def test_kineto(self):
         self.assertTrue(found_memcpy)
         # p.export_chrome_trace("/tmp/test_trace.json")
 
+    @unittest.skipIf(not kineto_available(), "Kineto is required")
+    @unittest.skipIf(not TEST_MULTIGPU, "Multiple GPUs needed")
+    def test_kineto_multigpu(self):
+        with profile(use_cuda=True, use_kineto=True) as prof:
```

Comment:
given that we want to deprecate torch.autograd.profiler, we should switch to torch.profiler.profile here?